src/css/reset.css

```diff
@@ -40,3 +40,11 @@ img {
 pre, code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New", monospace;
 }
+
+pre::-webkit-scrollbar {
+    display: none;
+}
+
+pre::scrollbar {
+    display: none;
+}
```

Comment:
We shouldn't disable the scrollbar display like this (although I agree with you that browser scrollbars are generally unattractive). Users depend on scrollbars to indicate that there's content below the fold. Hiding the scrollbar can be a confusing user experience. Also, unless users have a trackpad or a mouse-wheel, they would be unable to scroll the content in this case.

Reply:
Yes, correct :) Reverting this change and updating the PR.